spec/integration/fetching_linked_items_spec.rb

```diff
@@ -1,12 +1,13 @@
 require 'rails_helper'
+include ActiveSupport::Testing::TimeHelpers
 
 describe "Fetching linked items", type: :request do
   describe "GET /incoming-links/:base_path_without_root" do
     it "returns the items linked to an item" do
-      Timecop.freeze do
+      travel_to Time.now do
```

Comment:
I didn't at the time, but I did just now, and it looks like using a before block is no different (I still see a different timezone).

My suspicion is that the "wrong" value I am seeing is coming from Mongoid, which does use ActiveSupport for some time related things, so maybe it has better support for manipulating Mongoid than Timecop...